Git-Collabration-Example/ImageVC.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ImageVC.swift
+//  Git-Collabration-Example
+//
+//  Created by Jordan Peralta on 2022-06-15.
+//
+
+import UIKit
+
+class ImageVC: UIViewController {
+    
+    var getImage: UIImage?
+
+    @IBOutlet weak var showImage: UIImageView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        showImage.image = getImage
+//        showImage.image = UIImage(systemName: "star.fill")
```

Comment:
Can you please delete this line comment if not needed

Reply:
I will delete the unneccessary comment.